packages/app/cypress/e2e/settings.cy.ts

```diff
@@ -77,6 +77,23 @@ describe('App: Settings', () => {
         expect((ctx.actions.electron.openExternal as SinonStub).lastCall.lastArg).to.eq('http:/test.cloud/cloud-project/settings')
       })
     })
+
+    it('shows the recordKey when coming from a run', { retries: 0 }, () => {
+      cy.remoteGraphQLIntercept(async (obj) => {
+        await new Promise((resolve) => setTimeout(resolve, 100))
+
+        return obj.result
+      })
+
+      cy.startAppServer('e2e')
+      cy.loginUser()
+      cy.visitApp()
+      cy.get('.spec-list-container').scrollTo('bottom')
+      cy.contains('test1.js').click()
+      cy.get(`[href='#/settings']`).click()
+      cy.contains('Dashboard Settings').click()
+      cy.contains('Record Key')
+    })
```

Comment:
This PR touches 8 files and only adds one test, and the test doesn't specifically cover what changed in this PR, only a side-effect. @tgriesser what do you think about adding a test that the graphql ws survives even after the per-spec ws is closed?

Reply:
> and the test doesn't specifically cover what changed in this PR

Yeah I guess it depends on how you define that, it covers it in the sense that there's a failing test, and then changes which result in the test passing. To me that does cover the changes here, and in order to test the graphql ws surviving you need to restructure the code to make it "testable", which I'm not a huge fan of, particularly on the frontend where it's harder to `proxyquire` or something.

I'm curious what your approach to testing it might be?

Reply:
@flotwig ok, I renamed the test, made it a bit more robust against false positives, added better comments about what it's actually testing